src/axom/primal/geometry/BezierCurve.hpp

```diff
@@ -190,7 +287,10 @@ class BezierCurve
   /// Returns a copy of the Bezier curve's control points
   CoordsVec getControlPoints() const { return m_controlPoints; }
 
-  /// Reverses the order of the Bezier curve's control points
+  /// Returns a copy of the Bezier curve's control points
+  axom::Array<T> getWeights() const { return m_weights; }
```

Comment:
Do we want a deep copy of the weights?
(Looks like you were keying off how `getControlPoints()` was implemented a few lines up)

Reply:
You're right, I was just following the model of `getControlPoints()`, so I'm not sure if we want a deep copy of the weights. Does this one do something different than `getControlPoints()`? I would think that both return a deep copy.

Reply:
Thanks, I was questioning if either of these should be a deep copy. 
I think it's fine for now and we can change it in the future as necessary/desired.